persistence/sql/index.md

```diff
@@ -23,6 +23,19 @@ The SQL Persistence uses [Json.NET](http://www.newtonsoft.com/json) to serialize
 partial: supportedimpls
 
 
+### Supported object names lengths
```

Comment:
Or identifiers? Better suggestions welcome!

Reply:
Should be "name lengths" but I think that's sufficiently generic for all dialects.